.circleci/config.yml

```
@@ -9,10 +9,10 @@ jobs:
   build:
     machine: true
     working_directory: ~/StackStorm/st2chatops
-    parallelism: 4
+    parallelism: 5
     shell: /bin/bash --login
     environment:
-      DISTROS: xenial bionic el7 el8
+      DISTROS: xenial bionic el7 el8 focal
```

Can we change the order a bit to reflect the original st2-packages order? 

```suggestion
      DISTROS: xenial bionic focal el7 el8
```

So it will be easier to associate build container with the OS in CircleCI
https://app.circleci.com/pipelines/github/StackStorm/st2chatops/47/workflows/f96ad5c0-53fe-41ed-a12c-34113dcc3120/jobs/933